rocprim/include/rocprim/intrinsics/atomic.hpp

```diff
@@ -33,7 +33,7 @@ namespace detail
         #ifdef ROCPRIM_HC_API
             return hc::atomic_fetch_add(address, value);
         #else
-            return ::atomicAdd(address, value);
+            return atomicAdd(address, value);
```

Comment:
Why making sure we're calling `atomicAdd` from global namespace would cause problems?

Reply:
I thought the atomic functions in HIP were static, but I notice that isn't true

Reply:
How is that relevant?